src/operator/numpy/np_matrix_op-inl.h

```diff
@@ -487,12 +487,12 @@ void NumpyRollCompute(const nnvm::NodeAttrs& attrs,
   CHECK_EQ(req.size(), 1U);
   if (inputs[0].Size() == 0U) return;
   const NumpyRollParam& param = nnvm::get<NumpyRollParam>(attrs.parsed);
-  const index_t ndim(inputs[0].shape_.ndim());
+  const int ndim(inputs[0].shape_.ndim());
```

Comment:
so for LTS the number of dimensions in the array is still (2^31)-1, but the size of a dimension can be bigger than 2^32?

Reply:
Yes. Number of dims has always been int. Here the original author used index_t but there is no point in doing so. Plus I think we have a hard limit for any tensors to be <= 10 dims

Reply:
size of individual dims and total size can both be >= 2**31 when large tensor is on